sonic-net/sonic-buildimage
# SBOM of sonic-broadcom.bin built from master on 20240717

####Why I did it
To enhance the security posture by providing a comprehensive Software Bill of Materials (SBOM) for the SONiC image, ensuring better traceability and vulnerability management.

####How I did it
Generated an SBOM using industry-standard tools and integrated it into the build process of the SONiC image.

####How to verify it
Validate the presence of the SBOM file in the SONiC image build directory and ensure its contents accurately reflect the components used.

####Description for the changelog
Generated SBOM for sonic-broadcom.bin built from the master branch on 20240717.

Signed-off-by: Alex Levit <alex@bitfidence.com>
